docs/source/config/intro.rst

```diff
@@ -112,6 +112,18 @@ At present, this only affects the current session - changes you make to
 config are not saved anywhere. Also, some options are only read when
 IPython starts, so they can't be changed like this.
 
+.. _configure_embedded:
+
+Configuring embedded IPython
```

Comment:
'Embedding' has a particular meaning in IPython: when you call `IPython.embed()`, you can see the variables from the scope where it was called in the shell. `start_ipython()`, which your example uses, starts IPython normally without this. We encourage people to use `start_ipython()` if they don't need that, because it's more robust.

So maybe we should call this something like 'Configuration for launching IPython from Python code'? Better ideas welcome :-)